source/bin/aws-instance-scheduler.ts

```diff
@@ -14,34 +14,34 @@
  *  permissions and limitations under the License.                            *
  *****************************************************************************/
 
-import * as cdk from '@aws-cdk/core';
-import { AwsInstanceSchedulerStack } from '../lib/aws-instance-scheduler-stack';
-import { AwsInstanceSchedulerRemoteStack } from '../lib/aws-instance-scheduler-remote-stack';
-
-const SOLUTION_VERSION = process.env['DIST_VERSION'] || '%%VERSION%%';
-const SOLUTION_NAME = process.env['SOLUTION_NAME'] ? process.env['SOLUTION_NAME'] : "aws-instance-scheduler";
-const SOLUTION_ID = process.env['SOLUTION_ID'] ? process.env['SOLUTION_ID'] : "SO0030";
-const SOLUTION_BUCKET = process.env['DIST_OUTPUT_BUCKET'] ? process.env['DIST_OUTPUT_BUCKET'] : "";
-const SOLUTION_TMN = process.env['SOLUTION_TRADEMARKEDNAME'] ? process.env['SOLUTION_TRADEMARKEDNAME'] : "aws-instance-scheduler";
-const SOLUTION_PROVIDER = 'AWS Solution Development';
-
-const app = new cdk.App();
-
-new AwsInstanceSchedulerStack(app, 'aws-instance-scheduler', {
-    description: `(${SOLUTION_ID}) - The AWS CloudFormation template for deployment of the ${SOLUTION_NAME}, version: ${SOLUTION_VERSION}`,
-    solutionId: SOLUTION_ID,
-    solutionTradeMarkName: SOLUTION_TMN,
-    solutionProvider: SOLUTION_PROVIDER,
-    solutionBucket: SOLUTION_BUCKET,
-    solutionName: SOLUTION_NAME,
-    solutionVersion: SOLUTION_VERSION
-});
-new AwsInstanceSchedulerRemoteStack(app, 'aws-instance-scheduler-remote', {
-    description:  `(${SOLUTION_ID}S) - The AWS CloudFormation template for ${SOLUTION_NAME} cross account role, version: ${SOLUTION_VERSION}`,
-    solutionId: SOLUTION_ID,
-    solutionTradeMarkName: SOLUTION_TMN,
-    solutionProvider: SOLUTION_PROVIDER,
-    solutionBucket: SOLUTION_BUCKET,
-    solutionName: SOLUTION_NAME,
-    solutionVersion: SOLUTION_VERSION
-});
\ No newline at end of file
+ import * as cdk from 'aws-cdk-lib';
+ import { AwsInstanceSchedulerStack } from '../lib/aws-instance-scheduler-stack';
+ import { AwsInstanceSchedulerRemoteStack } from '../lib/aws-instance-scheduler-remote-stack';
+ 
+ const SOLUTION_VERSION = process.env['DIST_VERSION'] || '%%VERSION%%';
```

Comment:
cdk context file might be a better fit for these placeholders instead of environment variables, see [here](https://docs.aws.amazon.com/cdk/v2/guide/context.html)